CSS/css_completions.py

```diff
@@ -509,8 +509,8 @@ def on_query_completions(self, view, prefix, locations):
 
             for prop in self.props:
                 if add_colon:
-                    l.append((prop, prop + ": "))
+                    l.append((prop + "\tcss", prop + ": "))
```

Comment:
Shouldn't this rather get `"\tproperty"`?

Reply:
May other plugins provide completions within a CSS scope if the CSS is embedded into another syntax, which may or may not be meant for CSS but are displayed anyway and therefore might cause confusion?

With CSS in mind as a standalone syntax "property" was my first choice, too.

Reply:
I'm inclined to blame those other plugins for that. I don't use any third-party package for standard css or HTML, so I'm not running into any of this, but that doesn't mean we shouldn't consider third-party packages at all.